components/SSI/DIDxWallet/Wallet/Allowances/index.tsx

```diff
@@ -108,6 +144,7 @@ function Component() {
                     net === "mainnet" ? "" : "dev-"
                   }api.zilliqa.com`
                 );
+                updateDonation(null);
```

Comment:
it is ok to move this update to the end of the method, line 202?

Reply:
Yes, sure it was ok